src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/a2da67af29.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To Do list</title>
+  </head>
+  <body>
+    <main id="main">
+      <div>
+        <div class="todo-title flex">
+          <h1>Today's To Do</h1>
+          <i class="sync-btn fa-solid fa-rotate"></i>
+        </div>
+        <hr />
```

Comment:
Hey Ikoote, I would suggest you use the `<hr>` tag for the horizontal rule the forward slash was necessary for previous versions of HTML but it was removed in HTML 5